components/Header/index.js

```diff
@@ -1,12 +1,29 @@
 import styled from "styled-components";
+import Link from "next/link";
 
 export default function Header() {
-  return <StyledHeader>Hammerhart</StyledHeader>;
+  return (
+    <StyledContainer>
+      <StyledHeaderLink href="/">
+        <h1>⚒️ Hammerhart</h1>
+      </StyledHeaderLink>
+    </StyledContainer>
+  );
 }
 
-const StyledHeader = styled.h1`
-  color: white;
+const StyledContainer = styled.div`
   display: flex;
-  padding: 1rem;
   justify-content: center;
+  padding: 1rem;
+  position: fixed;
+  width: 100%;
+  top: 0;
+  z-index: 999;
+  background-color: #95786c;
```

Comment:
As a note for the future: Try to be consistent with the color naming (using hex or rgb). Maybe you can find some team conventions.